core/src/main/java/org/elasticsearch/index/query/DisMaxQueryBuilder.java

```diff
@@ -54,6 +61,24 @@ public DisMaxQueryBuilder add(QueryBuilder queryBuilder) {
     }
 
     /**
+     * Add a list of sub-queries to this disjunction.
+     */
+    public DisMaxQueryBuilder add(List<QueryBuilder> queryBuilders) {
```

Comment:
I think I wanted the setter to be similar to the must/should/mustNot setters in BoolQueryBuilder where are now able to add list. No problem to remove it though.